charts/kafka/charts/zookeeper/templates/statefulset.yaml

```diff
@@ -32,222 +34,198 @@ spec:
       release: {{ .Release.Name }}
       component: server
   updateStrategy:
-{{ toYaml .Values.updateStrategy | indent 4 }}
+    {{- toYaml .Values.updateStrategy | nindent 4 }}
   template:
     metadata:
       labels:
         app: {{ template "zookeeper.name" . }}
         release: {{ .Release.Name }}
         component: server
-      {{- if .Values.podLabels }}
+        {{- if .Values.podLabels }}
         ## Custom pod labels
         {{- range $key, $value := .Values.podLabels }}
         {{ $key }}: {{ $value | quote }}
-        {{- end }}
       {{- end }}
-{{- if .Values.podAnnotations }}
+      {{- end }}
+      {{- if .Values.podAnnotations }}
       annotations:
         ## Custom pod annotations
         {{- range $key, $value := .Values.podAnnotations }}
         {{ $key }}: {{ $value | quote }}
-        {{- end }}
-{{- end }}
+    {{- end }}
+    {{- end }}
     spec:
       terminationGracePeriodSeconds: {{ .Values.terminationGracePeriodSeconds }}
-{{- if .Values.schedulerName }}
+      {{- if .Values.schedulerName }}
       schedulerName: "{{ .Values.schedulerName }}"
-{{- end }}
+      {{- end }}
       securityContext:
-{{ toYaml .Values.securityContext | indent 8 }}
-{{- if .Values.priorityClassName }}
+        {{- toYaml .Values.securityContext | nindent 8 }}
```

Comment:
nindent is the same as indent but prepends a newline to the beginning of the string.
I would argue that indent is correct here.
Does it still make sense to have indent as you have indented the function by hand?